pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[project]
+name = "gha_runner"
+version = "0.0.1.dev0"
+description = "A GitHub Runner for deploying on cloud backends"
+authors = [{ name = "Ethan Holz", email = "ethan.holz@omsf.io" }]
+readme = "README.md"
+requires-python = ">= 3.10"
+dependencies = ["boto3", "PyGithub", "moto[ec2]", "pytest"]
```

Comment:
move testing deps to a `[project.optional-dependencies]` block. Last I checked, the most commonly used name was `test`, although I've also seen `tests` and `testing`.

Reply:
I am going to opt to use `test` to keep with best practices.